src/AccessibilityTranslator.tsx

```diff
@@ -1,8 +1,50 @@
 import _ from 'lodash';
-import React from 'react';
+import React, { ReactElement } from 'react';
 import PropTypes from 'prop-types';
 import { ROLES, getTagNameForDisplayObject } from './Roles';
 
+interface Props {
+  stage: object;
+}
+
+type ElementBounds = {
+  height?: number;
+  width?: number;
+  x: number;
+  y: number;
+};
+
+type AccessibleDisplayObject = createjs.DisplayObject & {
+  accessible?: {
+    _reactProps: {
+      key: string;
+      id: string;
+      onFocus: Function;
+      onBlur: Function;
+    };
+    children: AccessibleDisplayObject[];
```

Comment:
- in the original (investigation) PR, I had the wrong type here... root and all of its descendants (children) are of this object type